scripts/react_native_pods.rb

```diff
@@ -3,7 +3,12 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-def use_react_native! (options={})
+def resolve_module(request)
+  script = "console.log(path.dirname(require.resolve('#{request}/package.json')));"
+  Pod::Executable.execute_command('node', ['-e', script], true).strip
+end
+
+def use_react_native!(options={})
   # The prefix to the react-native
   prefix = options[:path] ||= "../node_modules/react-native"
```

Comment:
Do we want to use resolve_module here too? I think it would be better

Reply:
Here we need the paths to be relative to `Podfile`, otherwise CocoaPods won't be able to resolve the modules. There are ways to find the path to the `Podfile` but I don't think we should introduce that here.

Reply:
@Titozzz we always pass `options.path` by default in the template, which is absolute location of React Native as resolved by the CLI.